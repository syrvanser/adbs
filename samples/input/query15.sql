SELECT * FROM Sailors WHERE Sailors.B > 100 AND Sailors.B != Sailors.C AND Sailors.B <= 400 AND Sailors.A < 10 AND Sailors.A >= -1