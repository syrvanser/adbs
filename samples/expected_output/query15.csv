1,200,50
6,300,400